Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -0,0 +1,256 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using Xamarin.Forms.Platform;
+using static Xamarin.Forms.IndicatorView;
+
+namespace Xamarin.Forms
+{
+	public static class IndicatorViewExtensions
```

Comment:
I'd move this class to another file. It's odd to me that this is the first class I look at when I open `IndicatorView.cs`.

Reply:
@adrianknight89 agreed. It's a good point. But I don't know what could be the best place for this file

Reply:
We have a lot of extension classes in the Xamarin.Forms.Core project. Search for extensions in VS. You'll see them. You could place it in Xamarin.Forms or Xamarin.Forms.Internals namespace. There doesn't seem to be consistency among extension classes in terms of namespaces. @StephaneDelcroix can guide you better. 

Also some have `[EditorBrowsable(EditorBrowsableState.Never)]` while others don't. Hmm.

Reply:
I agree we can move this to internals name space and hide it. Can we also add a example of IndivatorsView binding to a VM without using carouselView so our users can understand they can use this without a CarouselVIew.

Thanks